src/azure-cli-core/azure/cli/core/_profile.py

```diff
@@ -193,9 +195,10 @@ def find_subscriptions_on_login(self,
             if is_service_principal:
                 if not tenant:
                     raise CLIError('Please supply tenant using "--tenant"')
-                sp_auth = ServicePrincipalAuth(password)
+                sp_auth = ServicePrincipalAuth(password, use_sn_issuer)
```

Comment:
What is "sn"?

Reply:
It means certificate subject name :). This abbreviation is used in the doc across, so i just use it. I know there is a risk it could be taken as serial number, so I used the full term in the command help.

Reply:
let me rename to `--use-cert-sn-issuer` to make it a bit more clear